docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/amazon-aws.md

```diff
@@ -456,7 +456,7 @@ cd ~
 pm2 start ecosystem.config.js
 ```
 
-Your Strapi project should now be available on `http://your-ip-address:1337/`.
+Your Strapi project should now be available on `http://your-local-ip-address:1337/`. Your local IP address will be that of your computer, rather than the one from the Ubuntu server.
```

Comment:
This isn't correct, it should be the ip address of the server, not your local machine.